packages/compass-collection/src/components/collection-header/collection-header.tsx

```diff
@@ -18,11 +18,12 @@ import TimeSeriesBadge from './time-series-badge';
 import ViewBadge from './view-badge';
 import CollectionStats from '../collection-stats';
 import type { CollectionStatsObject } from '../../modules/stats';
+import ClusteredBadge from './clustered-badge';
 
 const collectionHeaderStyles = css({
   paddingTop: spacing[3],
   paddingBottom: spacing[1],
-  minHeight: spacing[6] + spacing[1],
+  minHeight: +spacing[6] + +spacing[1],
```

Comment:
Do we need a min height here? I'm thinking letting the elements define the height and adding some bottom padding here would be a solution the might make the height not required to change if the elements shrink in new leafygreen iterations for instance?

Reply:
We need it for keeping consistent height when swapping between views and non-views as the elements on the right are a different height.